### DISCUSSION TITLE
Portainer Deployment inside a CasaOS in Single click @ Yesbhautik

### DISCUSSION THREAD

**Discussion by yesbhautik** [IceWhaleTech/CasaOS#1028] [Show and tell]
Step 1: Download the [.json file](https://go.yesbhautik.co.in/ParallelGreenMacaw) | OR | Make .json file with below code snippet
Step 2: Go to AppStore
Step 3: Click on Custom Install
Step 4: Click on import (top right corner | left side of the close button)
Step 5: Click on AppFile
Step 6: Upload .json file
Step 7: Click on Install (You can change the port)

Now, you can open your Firefox: localhost:8080 or ipv4:8080 (this is the default port)

![image](https://user-images.githubusercontent.com/63995315/234703918-b89e1645-5164-4578-abe7-f4323fae386e.png)

`{"host":"","protocol":"http","port_map":"8080","cpu_shares":90,"restart":"always","label":"portainer","position":false,"index":"","icon":"https://icon.casaos.io/main/all/portainer.png","network_model":"bridge","image":"portainer/portainer","description":"","origin":"local","ports":[{"container":"9000","host":"8080","protocol":"tcp","desc":"","type":0}],"volumes":[{"container":"/data","host":"/var/lib/docker/volumes/portainer_data/_data","type":0,"desc":""},{"container":"/var/run/docker.sock","host":"/var/run/docker.sock","type":0,"desc":""}],"envs":[{"container":"PATH","host":"/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin","desc":"","type":0}],"devices":[],"cap_add":[],"cmd":[],"privileged":false,"host_name":"6241b5961263","container_name":"portainer","appstore_id":0,"custom_id":"b925c68f-e6fa-45f2-a09f-a241d9c1a432","enable_upnp":false,"version":"1.0"}`

**Comment by zhanghengxin** [comment]
In version v0.4.4, you can now register your own store for users to use.
